modules/configuring-private-storage-endpoint-azure-vnet-subnet-iro-discovery.adoc

```diff
@@ -8,14 +8,14 @@
 
 The following procedure shows you how to set up a private storage endpoint on Azure by configuring the Image Registry Operator to discover VNet and subnet names.
 
-.Prerequisites 
+.Prerequisites
 
 * You have configured the image registry to run on Azure.
-* Your network has been set up using the Installer Provisioned Infrastructure installation method. 
+* Your network has been set up using the Installer Provisioned Infrastructure installation method.
```

Comment:
Would adding a link in an additional resources section be beneficial here? On docs.redhat.com, accessing the Installer document is not as rapid as docs.openshift.com.